### ISSUE TITLE
Wrap up the loss function

### ISSUE THREAD

**Issue by Yiming-M** [jia-wan/GeneralizedLoss-Counting-Pytorch#5]
Hi @jia-wan,

Thanks for your excellent work. Could you please also pack your loss into one function that only takes the predicted density map and the ground-truth as input arguments?

Many thanks,
Yiming
